src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoTest.java

```diff
@@ -281,13 +285,54 @@ public void partitionTest() throws Exception {
     }
   }
 
+  private void ingestOMOPTable(
+      Dataset dataset, String tableName, String ingestFile, int expectedRowCount) throws Exception {
+    List<Map<String, Object>> data;
+    try {
+      data = jsonLoader.loadObjectAsStream(ingestFile, new TypeReference<>() {});
+    } catch (Exception e) {
+      throw new RuntimeException("Error building ingest request", e);
+    }
```

Comment:
Since the exception will fail the test I don't see the value in catching and rethrowing an error here
```suggestion
    List<Map<String, Object>> data = jsonLoader.loadObjectAsStream(ingestFile, new TypeReference<>() {});
```

Reply:
Good point - I'll remove.